README.md

```diff
@@ -1,84 +1,275 @@
-# Node.js Toolkit
-
-#### The toolkit is a Node.js wrapper over the XMLSERVICE open source project from IBM. 
+# Node.js iToolkit <!-- omit in toc -->
 
 [![NPM](https://nodei.co/npm/itoolkit.png?downloads=true&downloadRank=true)](https://nodei.co/npm/itoolkit/)
 
-# Installation  
 
-Installation is done from a PASE shell.
+`itoolkit` is a Node.js interface to [XMLSERVICE](https://github.com/IBM/xmlservice) to access all things IBM i.
+
+# Table of Contents <!-- omit in toc -->
+- [Installation](#installation)
+- [Getting Started](#getting-started)
+  - [Main Classes](#main-classes)
+    - [Connection](#connection)
+      - [Transports](#transports)
+        - [idb-connector](#idb-connector)
+        - [REST](#rest)
+        - [SSH](#ssh)
+    - [ProgramCall](#programcall)
+      - [Example](#example)
+    - [CommandCall](#commandcall)
+      - [Example](#example-1)
+    - [SqlCall](#sqlcall)
+      - [Example](#example-2)
+  - [Utility Toolkit Functions](#utility-toolkit-functions)
+- [Documentation](#documentation)
+- [Testing](#testing)
+- [Contributing](#contributing)
+- [License](#license)
+
+# Installation
+
+Before installing, download and install Node.js
 
 ```sh
     $ npm i itoolkit
 ```
+# Getting Started
```

Comment:
There seems to be only one entry under "Getting Started", so I'd suggest getting rid of this altogether and moving everything up one level.

Reply:
Sounds good will remove